charts/alfresco-search-service/charts/alfresco-insight-zeppelin/templates/deployment.yaml

```diff
@@ -3,44 +3,56 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "alfresco-insight-zeppelin.fullName" . }}
+  name: {{ template "alfresco-insight-zeppelin.deployment.name" . }}
   labels:
-    app: {{ template "alfresco-insight-zeppelin.fullName" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-    component: zeppelin
+    {{- include "alfresco-insight-zeppelin.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app: {{ template "alfresco-insight-zeppelin.fullName" . }}
-      release: {{ .Release.Name }}
+      {{- include "alfresco-insight-zeppelin.selectorLabels" . | nindent 6 }}
   strategy:
     type: RollingUpdate
     rollingUpdate: {{- toYaml .Values.global.strategy.rollingUpdate | nindent 6 }}
   template:
     metadata:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/config.yaml") . | sha256sum }}
```

Comment:
please remove

Reply:
why? we have a configmap and should help if changing it